rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_info_interface.hpp

```diff
@@ -38,7 +38,7 @@ class ROSBAG2_STORAGE_PUBLIC BaseInfoInterface
    *
    * \returns the relative path.
    */
-  virtual std::string get_relative_path() const = 0;
+  virtual std::string get_relative_file_path() const = 0;
```

Comment:
This function makes more sense in the `io` interface, what do you think?

Reply:
I have an issue to move this #197 which I'm picking up after bagsplit merges.